ccd-definition/CaseField.json

```diff
@@ -377,5 +377,14 @@
     "FieldType": "MultiSelectList",
     "FieldTypeParameter": "DeletionConsent",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingDetails",
+    "Label": "Hearing ",
```

Comment:
Empty space after Hearing, please remove

Reply:
Not removed yet.

Reply:
This is because in the tabs I want it to appear Hearing 1 Hearing 2, without the space its Hearing1 Hearing2 no?

Reply:
I don't think that is correct @rebeccabrennan send me screenshots on slack